src/ixdat/data_series.py

```diff
@@ -235,3 +178,203 @@ def data(self):
                     f"{len(self._data.shape)}-dimensional."
                 )
         return self._data
+
+
+class ValueSeries(Field):
+    """Class to store scalar values that are measured over time.
+
+    Characterized by a reference to the corresponding time series. This reference is
+    represented in relational databases as a row in an auxiliary linker table
+    """
+
+    series_type = "vseries"
+
+    def __init__(
+        self,
+        name,
+        unit_name,
+        data,
+        t_id=None,
+        tseries=None,
+        a_ids=None,
+        axes_series=None,
+    ):
+        """Initiate a ValueSeries with a TimeSeries or a reference thereto
+
+        Args (in addition to those of parent):
+            t_id (int): The id of the corresponding TimeSeries, if not given directly
+            t_ids (list of int): [t_id], only so that a backend can pass t_id as a list
+            tseries (TimeSeries): The corresponding TimeSeries, if available
+        """
+        a_ids = a_ids or [t_id]
+        axes_series = axes_series or [tseries]
+        super().__init__(name, unit_name, data, a_ids, axes_series)
+        # TODO: This could probably be handled more nicely with PlaceHolderObjects
+        #   see: Measurement and
+        #   https://github.com/ixdat/ixdat/pull/1#discussion_r551518461
+
+    @property
```

Comment:
I've simplified this, getting rid of the hidden `tseries` and `t_ids`. It still has the hidden `axes_series` and `a_ids` though, so that it can be initiated with either an actual tseries (which becomes `_axes_series[0]` OR with a `t_id` (which becomes `a_ids[0]`) and in the latter case the actual `tseries` is loaded the first time something asks for it.